modules/microshift-greenboot-dir-structure.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/microshift-greenboot.adoc
+
+:_content-type: CONCEPT
+[id="microshift-greenboot-dir-structure_{context}"]
+= How greenboot uses directories to run scripts
+//Q: Are these correct for MicroShift? If not, what is the file structure and associated behavior we use?
+Health check scripts are available in read-only directories in RPM-OSTree systems. Four directories run greenboot scripts. Within each directory, scripts run in order of filename. Keep this in mind when you configure them for your workloads.
+
```

Comment:
"Scripts run in order of filename"? You mean alphabetical order?

Reply:
Is there a better way to state in which order in which files are read? How detailed do we need to be?